<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>603</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="Main_widget">
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="2" column="1">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>118</width>
        <height>119</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="5" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>138</width>
        <height>101</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="Bumpers">
      <property name="title">
       <string>Bumpers</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QCheckBox" name="RB_radio">
         <property name="text">
          <string>Right Bumper</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="LB_radio">
         <property name="text">
          <string>Left Bumper</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" rowspan="3">
     <widget class="QGroupBox" name="Left_stick">
      <property name="title">
       <string>Left Stick</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QSlider" name="ls_hslider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="ls_vdsb"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="ls_vlabel">
         <property name="text">
          <string>Vertical</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="ls_hdsb"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ls_hlabel">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="5">
        <widget class="QSlider" name="ls_vslider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="2" rowspan="2">
     <widget class="QGroupBox" name="Face_buttons">
      <property name="title">
       <string>Face Buttons</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QCheckBox" name="x_radio">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QCheckBox" name="a_radio">
         <property name="text">
          <string>A</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="b_radio">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="y_radio">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QGroupBox" name="Triggers">
      <property name="title">
       <string>Triggers</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="1">
        <widget class="QDoubleSpinBox" name="trig_dsb"/>
       </item>
       <item row="0" column="2" alignment="Qt::AlignRight">
        <widget class="QLabel" name="rtrig_label">
         <property name="text">
          <string>Right Trigger (Z-)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="ltrig_label">
         <property name="text">
          <string>Left Trigger (Z+)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="1" rowspan="2">
     <widget class="QGroupBox" name="Center_Buttons">
      <property name="minimumSize">
       <size>
        <width>141</width>
        <height>71</height>
       </size>
      </property>
      <property name="title">
       <string>Center Buttons</string>
      </property>
      <widget class="QCheckBox" name="back_radio">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>82</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="start_radio">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>30</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="2" rowspan="2">
     <widget class="QGroupBox" name="Right_stick">
      <property name="title">
       <string>Right Stick</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="rs_hlabel">
         <property name="text">
          <string>Horizontal</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QSlider" name="rs_hslider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QDoubleSpinBox" name="rs_hdsb"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="rs_vlabel">
         <property name="text">
          <string>Vertical</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QDoubleSpinBox" name="rs_vdsb"/>
       </item>
       <item row="0" column="1" rowspan="5">
        <widget class="QSlider" name="rs_vslider">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QGroupBox" name="Dpad">
      <property name="title">
       <string>Directional Pad</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="2">
        <widget class="QCheckBox" name="east">
         <property name="text">
          <string>E</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QCheckBox" name="northeast">
         <property name="text">
          <string>NE</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="north">
         <property name="text">
          <string>N</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QCheckBox" name="northwest">
         <property name="text">
          <string>NW</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="west">
         <property name="text">
          <string>W</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="southwest">
         <property name="text">
          <string>SW</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="south">
         <property name="text">
          <string>S</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QCheckBox" name="southeast">
         <property name="text">
          <string>SE</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="origin">
         <property name="text">
          <string>O</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="6" column="0" colspan="3">
     <widget class="QLineEdit" name="control_name_field"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>603</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
